test/format_entries_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'minitest/autorun'
+require 'minitest/mock'
+require 'tool_belt'
+require 'mocha/minitest'
+
+class FormatEntriesTest < MiniTest::Test
```

Comment:
Love seeing a test. What do you think about changing this from `FormatEntriesTest` to `ChangelogTest` to broaden the scope a bit, and keep future tests for the class in one file?